### PR TITLE
doc: refactor and improve how to contribute

### PR DESCRIPTION
**Pull request by kwy95**
This fixes some mistakes that were on the _How to contribute_ page.
It also uses the graphviz package to create the dev cycle figure using the DOT language.

### CODE REVIEW

File: documentation/content/howtocontribute.rst [hunk 1]
```diff
@@ -126,41 +126,41 @@ below Developer's Certificate of Origin (D-C-O):
 
 then you just add a line saying::
 
- Signed-off-by: Xpto Lalala Blabla <xpto@developer.example.org>
+  Signed-off-by: Jane Doe <doe.jane@developer.example.org>
```
Comment (joaoseckler) [inline documentation/content/howtocontribute.rst:128]: I liked the original brazilian version!

**Comment by rodrigosiqueira** [comment]
I really like it :)

Few comments:
1. Move the diagram to the same level as the codestyle file.
2. Delete the old image.
3. Could you try to reduce this page size a little bit? (just a little bit hahah)

**Comment by kwy95** [comment]
Thanks, will work on it!

Do you want it physically smaller, or do you want me to reword the text to make it shorter?
I assume it is the latter.

**Comment by rodrigosiqueira** [comment]
Reword some part of the text :)
Don't worry too much about that; just try to make some info more straightforward. You can also remove some URLs by using links in a word or phrase.

**Comment by rodrigosiqueira** [comment]
@kwy95 I think it is also a good idea to create a section that describes our Github actions, and make clear to everyone that they need to pass in everything to get their change applied. Finally, I think you can recommend people to read the Bash documentation (add a link) and Koalaman wike (https://github.com/koalaman/shellcheck/wiki).

**Comment by rodrigosiqueira** [comment]
It lgtm. Rebase it in the latest version from unstable to check if Travis triggers the automatic build.

**Comment by rodrigosiqueira** [comment]
@kwy95 any update here?

**Comment by rodrigosiqueira** [comment]
Merged, Thanks.
See: https://github.com/kworkflow/kworkflow/commit/a7f0438e1c9a174cd26b8041a20bf92c2c1e9d13

**Pull request closed without merging**